query/src/sql/planner/binder/select.rs

```diff
@@ -55,11 +60,52 @@ impl Binder {
         // Output of current `SELECT` statement.
         let mut output_context = self.normalize_select_list(&stmt.select_list, &input_context)?;
 
+        self.analyze_aggregate(&output_context, &mut input_context)?;
+
+        if !stmt.group_by.is_empty() {
```

Comment:
What's happen with `SELECT COUNT() FROM table`.

In this case, we don't seem to create an AggregatePlan

Reply:
Yes, in the ticket I don't consider the case.